float32 roverDistance
float32 roverBearing
